examples/authentication/sense-using-certificates/README.md

```diff
@@ -25,3 +25,14 @@ code example).
 
 * [Qlik Sense Help: Exporting certificates](http://help.qlik.com/en-US/sense/June2017/Subsystems/ManagementConsole/Content/export-certificates.htm)
 * [Qlik Sense Help: Certificate architecture](http://help.qlik.com/en-US/sense/June2017/Subsystems/PlanningQlikSenseDeployments/Content/Deployment/Server-Security-Authentication-Certificate-Trust-Architecture.htm)
+
+## PROXY
+
+To use engima when working behind a proxy. Set the operating system environment variable called `HTTPS_PROXY` to specify Proxy URL.
+
+export https_proxy=http://172.17.0.1:3128
```

Comment:
Think we should avoid IP addresses in the readme. Perhaps better to just state `export https_proxy=http://<ip address>:<port>`